.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# As a security feature, add these people as reviewers if a PR is created for
+# anything under .github, including GitHub Actions and this file itself.
+.github/ @jhmccoll @josekudiyirippil @gil0109
```

Comment:
We could alternatively set up a "Team" instead of individual member, but I thought we should try this out as a minimal implementation and we can expand later if it works for us.